stock_restrict_lot/models/stock_move.py

```diff
@@ -88,3 +89,44 @@ def _split(self, qty, restrict_partner_id=False):
         if vals_list and self.restrict_lot_id:
             vals_list[0]["restrict_lot_id"] = self.restrict_lot_id.id
         return vals_list
+
+    def get_all_dest_moves(self):
```

Comment:
@Kev-Roche Have you ever envisaged `_rollup`functions instead ? 

https://github.com/odoo/odoo/blob/16.0/addons/stock/models/stock_move.py#L2173

Reply:
Never... I was not aware of these functions, I change the code accordingly. Thank you @rousseldenis

Reply:
As discuss with @metaminux , we need a _rollup_not_cancelled_move_origs() method to take account of the done move.